tests/cql/CqlDateTimeOperatorsTest.xml

```diff
@@ -1177,11 +1183,11 @@
 		</test>
 		<test name="DateTimeSubtract2YearsAsMonths">
 			<expression>DateTime(2014) - 24 months</expression>
-			<output>@2012</output>
+			<output>@2012T</output>
 		</test>
 		<test name="DateTimeSubtract2YearsAsMonthsRem1">
 			<expression>DateTime(2014) - 25 months</expression>
-			<output>@2011</output>
+			<output>@2012T</output>
```

Comment:
See [CQL spec on DateTime subtraction](http://build.fhir.org/ig/HL7/cql/09-b-cqlreference.html#subtract-1), which states that decimals are dropped after converting the 2nd argument to the units of the first -- and also provides this example, which is very similar to the test above:
> DateTime(2014) - 18 months = DateTime(2013)